Library/Homebrew/version.rb

```diff
@@ -343,37 +341,27 @@ def self.detect(url, **specs)
     parse(specs.fetch(:tag, url), detected_from_url: true)
   end
 
+  # TODO: Deprecate this and just use `Version.new`.
```

Comment:
```suggestion
  # TODO: odeprecate this and just use `Version.new`.
```

and then it'll be caught on next minor version